react/features/chat/components/web/ChatInput.js

```diff
@@ -63,7 +68,8 @@ class ChatInput extends Component<Props, State> {
 
     state = {
         message: '',
-        showSmileysPanel: false
+        showSmileysPanel: false,
+        width: 0
```

Comment:
@mike-gusev-ae  I guess you can use 
```suggestion
        width: window.innerWidth
```

Reply:
but if `_onToggleSmileysPanel` always fires on open there is no reason to change that

Reply:
@mike-gusev-ae but if _onToggleSmileysPanel not fires in some `chat always open on init` it will be useful to change that line.